README.md

```diff
@@ -8,7 +8,7 @@ A basic AWS IoT MQTT test client that publishes randomized messages according to
 
 ### Documentation and Usage
 
-First, configure an AWS IoT Core Thing with associated security artifacts.  See instructions in [this AWS IoT tutorial](https://docs.aws.amazon.com/iot/latest/developerguide/create-iot-resources.html) for an example.  
```

Comment:
@mwengren most of the pre-commit failures can be "fixed" automatically by running `pre-commit run --all-files`. This is what I did here. If you think that there is a lot of churn in the code we can remove this commit.